### PR TITLE
done with chapter 4 and 5

### PR DESCRIPTION
**Pull request by sim981997**
Ready

### CODE REVIEW

**Comment by sim981997** [comment]
Ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.0'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
```
Comment (manhnt-0960) [inline Gemfile]: plesea remove all comment

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
```
Comment (manhnt-0960) [inline Gemfile]: use `"` instead of `'`

File: app/assets/stylesheets/custom.scss [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+margin-bottom: 30px;
+text-align: center;
+font-weight: normal;
+color: $light-gray;}
+
+/* header */
+
+#logo {
+float: left;
+margin-right: 10px;
```
Comment (manhnt-0960) [inline app/assets/stylesheets/custom.scss:46]: indent 2 spaces

File: app/assets/stylesheets/static_pages.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the static_pages controller here.
```
Comment (manhnt-0960) [inline app/assets/stylesheets/static_pages.scss]: delete this file because u don't use

File: app/assets/stylesheets/users.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```
Comment (manhnt-0960) [inline app/assets/stylesheets/users.scss]: semilar

File: app/controllers/static_pages_controller.rb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+class StaticPagesController < ApplicationController
+	def home
+	end
```
Comment (manhnt-0960) [inline app/controllers/static_pages_controller.rb]:
```suggestion
    def home; end
```

File: app/controllers/users_controller.rb [hunk 7]
```diff
@@ -0,0 +1,4 @@
+class UsersController < ApplicationController
+  def new
```
Comment (manhnt-0960) [inline app/controllers/users_controller.rb]: semilar

File: app/helpers/application_helper.rb [hunk 8]
```diff
@@ -0,0 +1,11 @@
+module ApplicationHelper
+# Returns the full title on a per-page basis.
+def full_title(page_title = '')
```
Comment (manhnt-0960) [inline app/helpers/application_helper.rb]:
```suggestion
def full_title page_title
```

File: app/helpers/application_helper.rb [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+base_title = "Ruby on Rails Tutorial Sample App"
```
Comment (manhnt-0960) [inline app/helpers/application_helper.rb:4]: check indent 2 spaces, please

**Comment by sim981997** [comment]
Ready

**Comment by manhnt-0960** [comment]
please fix 2 commits

**Comment by manhnt-0960** [comment]
commented

**Pull request merged**